test/check-machines-networks

```diff
@@ -893,9 +914,55 @@ class TestMachinesNetworks(VirtualMachinesCase):
         # Check that all networks are there
         b.wait_in_text("body", "Networks")
         self.waitNetworkRow("default")
+        self.waitNetworkRow("test_network_ipv6")
 
         # Expand row for first network
         self.toggleNetworkRow("default")
+        # Expand the network created above
+        self.toggleNetworkRow("test_network_ipv6")
+
+        # "Add entry" is not supported for IPv6 network
+        b.wait_not_present("#network-test_network_ipv6-system-static-host-entries-add")
+        b.wait_in_text("#network-test_network_ipv6-system-ipv6-dhcp-static", "none")
+
+        # Check "X" button of the dialog, "Cancel" is checked when xfail set
+        b.click("#network-default-system-static-host-entries-add")
+        b.wait_visible("#add-new-static-entry")
+
+        b.click("#add-new-static-entry button[aria-label=Close]")
+        b.wait_not_present("#add-new-static-entry")
+
+        # Check the errors when the all inputs are empty
+        d = NetworkAddStaticDCHPHostsDialog(
+            self,
+            xalert=True,
```

Comment:
Anyway - you can ignore this comment and we can get this in, but I would like that we create some test pattern for checking the helper messages, rather than doing different for each dialog. Can be followup.

Reply:
OK, will do that